employees/common/exports.py

```diff
@@ -74,6 +84,24 @@ def set_active_worksheet_name(workbook: Workbook, author: CustomUser) -> Workshe
     return worksheet
 
 
+def get_report_date_and_daily_hours(reports_date: List[str], date: str, reports: Report):
```

Comment:
Looks difficult, how about adding some comment(s) to this function explaining what is going on? :)

Reply:
Sure, I'm going to add comment to this :)